misc/git.talon

```diff
@@ -59,15 +58,26 @@ git reset soft: "git reset --soft "
 git reset hard: "git reset --hard "
 git restore: "git restore "
 git restore staged: "git restore --staged "
-git remote show origin: "git remote show origin\n"
+git restore source: "git restore --source="
+git remote: "git remote "
+git remote add: "git remote add "
 git remote add upstream: "git remote add upstream "
+git remote remove: "git remote remove "
+git remote show origin: "git remote show origin\n"
 git show: "git show "
 git stash pop: "git stash pop\n"
 git stash: "git stash\n"
 git stash apply: "git stash apply\n"
 git stash list: "git stash list\n"
 git stash show: "git stash show"
 git status: "git status\n"
+git switch [<user.text>]: "git switch {user.formatted_text(text or '', 'DASH_SEPARATED')}"
+git switch master: "git switch master "
+git switch main: "git switch main "
```

Comment:
Could maybe include `develop` if you're including `master` and `main`

Reply:
what's the point of having explicit master and main if they're already covered by the first rule that uses `[<user.text>]`? `develop` will also already be covered by it.

Reply:
Just to get talon to favour these I would think.  I'd consider putting an `enter` on these two tho maybe that's a bit aggressive?

Reply:
Makes sense about making talon favor it, may as well keep it then. I use \n for checkouts on my repo already iirc so I'm fine with adding them. I guess sometimes people will legitimately want to add arguments after in which case it will be a bit harder to use, but I suspect that's quite rare.